zqd/handlers.go

```diff
@@ -248,14 +248,22 @@ func handlePacketPost(c *Core, w http.ResponseWriter, r *http.Request) {
 		select {
 		case <-proc.Done():
 			done = true
+		case <-proc.Snap():
 		case <-ticker.C:
```

Comment:
Hmm, it would be nice for the `ticker` to be reset here, though I don't see away to do this with the existing `time.Ticker` api.

Reply:
Yeah. But worst case we'll send a handful of redundant status messages.